### PR TITLE
Gtharris dev

### PR DESCRIPTION
**Pull request by gtharris**
#### Summary of Changes
I pulled off a lot of my changes into a seperate forked repo because I was triggering jonny's build too much. I have no idea why the build has suddenly stopped working ever since we went private shortly. 

#### Build Status
TODO:  Replace YOUR_BRANCH_HERE - [Build Status](https://travis-ci.org/jonnykry/boxD)

### CODE REVIEW

File: server/server.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import os
+from google.appengine.ext import tor
```
Comment (samjxn) [inline server/server.py]: I'm thinking this line can be removed?

**Review by samjxn — CHANGES_REQUESTED** [review]

File: server/server.py (hunk 1 above)
Comment (gtharris) [reply server/server.py]: removed

File: server/server.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import os
```
Comment (samjxn) [inline server/server.py:1]: I should have mentioned on my last comment that the `os` package is needed (see line 27) 😐

**Review by samjxn — CHANGES_REQUESTED** [review]

**Comment by jonnykry** [comment]
:shipit: after Sam's comment is addressed

**Comment by samjxn** [comment]
:shipit:

**Pull request merged**